ci/run-check.sh

```diff
@@ -45,4 +45,19 @@ case $TARGET in
 
   docs-build)
     RUSTDOCFLAGS="-D warnings" cargo doc --all --no-deps
+    ;;
+  subalfred)
+    # Find all child directories containing Cargo.toml files
+    # TODO: Filter by crates found in the workspace
+    #   HINT: Use `cargo workspaces list -l" and filter by the paths
```

Comment:
Do we wanna address this or remove the hint?

Reply:
I leave these comments in order to tackle them in another PR. Is it ok for you?